modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -264,9 +264,9 @@
                 view-nm))
            (let [table-id (db/select-one-id Table :db_id (u/the-id database), :name view-nm)]
              ;; and its columns' :base_type should have been identified correctly
```

Comment:
Note that this version of the driver now detects "hello" as `character varying(5)` vs. plain `varchar`. IDK how this affects the linked issue at https://github.com/metabase/metabase/issues/21215 and fix at https://github.com/metabase/metabase/pull/24910. It sounds like the original problem was the driver didn't pick up the length, but now it does. Looks like @noahmoss's PR for #24910 may deal with this difference as well.

Reply:
I don't actually know much about redshift so I'll defer to @camsaul